nautobot/core/tests/test_commands.py

```diff
@@ -57,6 +58,11 @@ def test_success(self):
         out, err = self.run_command()
 
         self.assertIn("All pre-migration checks passed.", out)
+        # Assert Permission constrain warning not logged
+        self.assertNotIn(
+            f"ObjectPermission '{self.obj_perm.name}' (id: {self.obj_perm.pk}) has a constraint that references a model (nautobot.extras.models.datasources.GitRepository) that will be migrated to a new model by the Nautobot 2.0 migration.",
+            out,
+        )
```

Comment:
This seems like a really specific string for a negative test. Should it instead just be something like `self.assertNotIn(f"ObjectPermission '{self.obj_perm.name}'", out)` to catch *any* unexpected message for this permission?